front/app/modules/commercial/matomo/actions.test.ts

```diff
@@ -60,6 +52,7 @@ describe('trackPageChange', () => {
   });
 
   it('sets project id path is idea', async () => {
+    queryClient.setQueryData(ideasKeys.itemSlug('some-idea'), mockIdea);
```

Comment:
I guess this is already covered in the `beforeEach` block above?

Reply:
Good catch!